### ISSUE TITLE
ML: MP3/OGG stream to browser

### ISSUE THREAD

**Issue by modulesio** [webmixedreality/exokit#557]
We already support streaming ML scene composition from device to browser, but it would be good to add audio with that (if we continue with that streaming mode as opposed to RTMP).

FFmpeg could be used to encode the stream and we can read it client side as audio buffers.

Alternatively, this could be done as PCM but the bandwidth for that would be considerably more and it might end up being slower.

**Comment by modulesio** [comment]
RTMP does have an audio stream, it's just null right now. I agree it should probably have this same path.

**Comment by machenmusik** [comment]
so i guess the idea would be to make another input stream that is the composite output alongside the voice comm in https://github.com/modulesio/LabSound/blob/master/labsound/src/internal/src/ml/AudioDestinationMl.cpp, and wire it up to audio encoder and websocket / RTMP ?

**Comment by modulesio** [comment]
Yes. Conveniently I think the MLSDK has an endpoint for getting the mixed audio stream.
